src/CheckoutSdk/Payments/Links/PaymentLinkDetailsResponse.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Checkout.Common;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace Checkout.Payments.Links
+{
+
+    public sealed class PaymentLinkDetailsResponse : Resource, IEquatable<PaymentLinkDetailsResponse>
+	{
+
+		public string Id { get; set; }
+
+		public PaymentLinkStatus? Status { get; set; }
+
+		[JsonProperty(PropertyName = "expires_on")]
+		public string ExpiresOn { get; set; }
+
+		[JsonProperty(PropertyName = "return_url")]
```

Comment:
Please avoid `JsonProperty` if it's not needed. We did it like this in Java, but definitely is something we can get rid of it on this project.